setup.sh

```diff
@@ -0,0 +1 @@
+pipenv run flask db upgrade
```

Comment:
if it's a shell script then please add `#!/bin/sh` at the top and change permissions to 755. Also it's missing an end line.

Reply:
done :+1: